modules/Tasks.js

```diff
@@ -0,0 +1,24 @@
+class Task {
+  constructor(description, completed, index, editable) {
+    this.description = description;
+    this.completed = completed || false;
+    this.index = index;
+    this.editable = editable || false;
+  }
+
+  getHtml() {
+    return ` <li class="task" draggable="true">
+            <div class="description">
+         <input type="checkbox" class="check"  id="checkbox-${this.index - 1}" ${this.completed ? 'checked' : ''}/>
+         ${!this.editable ? `<p class="text">${this.description}</p>` : ''}
+         ${this.editable ? `<input value='${this.description}' id="input-${this.index - 1}"/>` : ''}</div>
+      <div id="myLinks-${this.index - 1}" style="display:none" class="list" >
+      <a href="#" class='del' id="del-${this.index - 1}">Delete</a>
+      <a href="#" class='edit'>Edit</a>
+    </div>
+      <button class="remove" id="remove-${this.index - 1}"><i class="fa-solid fa-ellipsis-vertical ellips"></i></button>
+      </li>`;
+  }
+}
+
```

Comment:
- [ ] Great work, your work is fine, however, you should consider changing one more thing in your readme file, make sure to insert your name instead of writing `@githubhandler` you can write something like `@ciraganenicole`, let's go and fix that, your project will be good to go ⚒️ 

<img width="389" alt="readme" src="https://user-images.githubusercontent.com/79058364/160241524-23c08100-6352-4bd7-96a3-6e2389757a08.png">